l10n_br_fiscal/models/res_company.py

```diff
@@ -253,6 +253,11 @@ def _compute_simplified_tax(self):
         comodel_name="l10n_br_fiscal.icms.regulation", string="ICMS Regulation"
     )
 
+    icms_difal_regulation_id = fields.Many2one(
+        comodel_name="l10n_br_fiscal.icms.difal.regulation",
+        string="ICMS Difal Regulation",
+    )
+
```

Comment:
@mileo você diz por um default aqui certo ?

cc @felipemotter

Reply:
Pode ser ai ou nos dados, mas fiquei um pouco na dúvida de como icms_regulation_id da empresa principal esta sendo preenchida.

Se for possível, quanto menos dados forem necessários para configurar uma empresa para operar melhor.

Reply:
@felipemotter @antoniospneto alguns dias atrás atualizando uma base no meu ambiente dev (com esta PR) deu erro devido a falta de valor em icms_difal_regulation_id. 

Infelizmente não printei mas acho que pode ser que tenha chamado o _compute_icms e caiu no teste da linha 425